.cicd/tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+fold-execute mkdir -p $BUILD_DIR
+
+TEST_COMMAND="fold-execute ctest -j$JOBS --output-on-failure -T Test"
```

Comment:
What is the advantage to making these BASH commands a variable that is then evaluated for execution?
Let's hard-code the ctest commands, because they should not be changing and they aren't meant to be configurable.

Reply:
I disagree. I've seen them change multiple times since I started here.